server/api/controllers/sermons/get-all-announcements.js

```diff
@@ -0,0 +1,48 @@
+const { DateTime } = require('luxon');
+const he = require('he');
+
+const transformAnnouncement = (announce) => {
+  return {
+    id: announce.id,
+    slug: announce.slug,
+    title : he.decode(announce.title.rendered),
+    startDate: DateTime.fromFormat(announce.date, "LLLL MM dd yyyy"),
```

Comment:
any reason why the formats for the time are different? ideally it should just be `2021-07-04T21:24:03.000+08:00` as an example

Reply:
that's the one im not rly sure, cuz the json file format was in unicode like "Fri \u00b7 7\/30\/2021" not sure whats the format for this...